src/modules/75-cd/components/PipelineStudio/DeployServiceSpecifications/SelectDeploymentType.tsx

```diff
@@ -340,7 +340,7 @@ export default function SelectDeploymentType(props: SelectServiceDeploymentTypeP
         deploymentType: getServiceDeploymentTypeSchema(getString)
       })}
     >
-      {formik => {
+      {(formik: any) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232293)
```suggestion
      {(formik: unknown) => {
```